---
layout: page
title: Current Research
permalink: /current_research/
---

## <span style="color:brown"> RETROVIRUS-HOST INTERACTIONS AND INNATE IMMUNITY TO GENE TRANSFER </span>

Our research integrates Bioinformatics and Molecular Biology to investigate the complex interplay between gene editing vectors, host cells, and the immune system, with a focus on advancing gene therapy strategies.  
Leveraging different -omics data such as single-cell RNA sequencing (scRNAseq), bulk RNA sequencing (RNAseq), proteomics and metabolomics, we focus on three broad projects: adeno-associated virus (AAV) vector-based therapies, inflammatory diseases, and the biology of quiescent human hematopoietic stem cells. We aim to elucidate innate immune sensing mechanisms, particularly the roles of interferon-inducible transmembrane proteins (e.g., IFITM3) in antiviral defenses and the regulation of endosomal immune responses. Our recent findings include how interferon-regulated phospholipids influence endosomal immunity and how gene-editing vectors, such as AAV, interact with cellular pathways and immune restrictions.

- In the study of inflammatory diseases like STING-associated vasculopathy with onset in infancy (SAVI) and Aicardi-Gouti√®res syndrome (AGS), we utilized scRNAseq and proteomics analyses to uncover molecular drivers of interferon-driven pathologies. For AGS, which mimics congenital viral infections, our proteomics and transcriptomics analyses highlighted key pathways involved in type I interferon dysregulation and identified potential targets for RNA-based therapies and inhibitors. These integrative approaches provided a comprehensive view of the genetic and proteomic landscape, advancing our understanding of inflammatory pathogenesis and therapeutic interventions.

- Our work on AAV-mediated gene therapy emphasizes the safety and efficacy of these vectors in neurological disorders. Using bulk RNAseq, scRNAseq and single-nucleus RNAseq (snRNAseq), in human and mouse stem cell-derived central nervous system (CNS) models, we demonstrated that AAV vector genomes activate p53-dependent DNA damage responses, leading to STING- and IL-1R-mediated inflammatory pathways. These findings highlight potential neurotoxicity risks and inform strategies to mitigate inflammatory responses, ensuring safer application of AAV therapies for neurological diseases.

- In studying quiescent human hematopoietic stem cells, we employed scRNAseq and bulk RNAseq analyses to dissect the molecular mechanisms underlying genomic stability and stress responses. Our analyses revealed critical pathways that maintain stem cell integrity under stress, providing insights into the biology of quiescence and its implications for regenerative medicine.

By combining advanced bioinformatics tools with experimental approaches, our work bridges computational and experimental biology to optimize gene therapy and advance therapeutic solutions for genetic and immune-mediated disorders.


---


## <span style="color:brown"> HUMAN HEMATOPOIETIC DEVELOPMENT AND DISEASE MODELING </span>

Our research focuses on leveraging pluripotent stem cells to model human hematopoietic development, aiming to elucidate the molecular mechanisms underlying blood cell production and hematopoietic stem cell specification.

A central aspect of our work involves the study of CD32 (FCGR2B) as a definitive marker for hemogenic endothelial cells (HECs). By applying advanced bioinformatics approaches, including transcriptomic profiling, developmental trajectory, and in-silico perturbation analyses, we have identified CD32 as a key marker specific to HECs during human embryonic development. Our findings demonstrate that CD32 expression is enriched in endothelial cells transitioning toward hematopoiesis and a functional validation confirmed that CD32+ HECs possess robust multilineage hematopoietic potential. Additionally, our analyses revealed that CD32+ HECs can generate hematopoietic progenitors independently of Notch signaling, providing a targeted approach for isolating these cells and enhancing the efficiency of in vitro blood cell production.

We also investigate the genetic basis of hematopoietic disorders, including Omenn syndrome (OS). We aim to dissect how dysregulated T cell differentiation and localization contribute to OS. Specifically, we utilize single-cell RNA sequencing (scRNAseq) and spatial transcriptomics to study the different roles of embryonic and adult T cells in OS pathogenesis and the T cell dynamics and inflammatory aspects in the colon. This computational approach offers a comprehensive framework to better understand hematopoietic development and inform potential therapeutic strategies in regenerative medicine and cell therapy.


---


## <span style="color:brown"> AUTOPHAGY ROLE IN COLORECTAL CANCER PROGRESSION AND THERAPY RESISTANCE </span>

We employ a sophisticated computational and data-driven experimental approach to investigate the role of AP3M2 in colorectal cancer (CRC). Initially, we have mined large-scale cancer genomics databases, such as The Cancer Genome Atlas (TCGA), c-BioPortal, TIMER, and GEPIA, to identify potential oncogenes and to understand the molecular underpinnings of CRC. Using these platforms, we have identyified and performed comprehensive bioinformatics analyses to explore the expression patterns, mutations, and copy number alterations of AP3M2 across various CRC samples. We further correlated AP3M2 expression with key autophagy markers (such as ATG7, LC3B, and P62) and analyzed its potential prognostic value through survival analyses, revealing significant associations between AP3M2 alterations and patient outcomes.  
To validate these computational findings, we employed RNA interference (RNAi) to knock down AP3M2 expression in CRC cell lines, including HCT116, HT29, and CACO2. The impact of AP3M2 silencing on CRC cell viability, migration, clonogenicity, and anchorage-independent growth was assessed through various in vitro assays. Additionally, bioinformatic tools were used to analyze ROS production and its relationship with autophagy in the context of AP3M2 knockdown. The combined use of genomic data mining, bioinformatics analysis, and experimental validation demonstrates a multi-faceted approach, revealing AP3M2 as a key modulator of CRC progression through its regulation of autophagy and ROS levels. This approach highlights the importance of computational strategies in guiding experimental research and identifying promising therapeutic targets.


