---
layout: resume
title: Curriculum vitae
permalink: /CV/
---

## Personal Information

- <span style="color:brown"> _Address:_ </span> Via Adolfo Ferrata 17, Pavia 27100 (PV), Italy
- <span style="color:brown"> _Email:_ </span> aboualezz.monah@hsr.it
- <span style="color:brown"> _Portfolio:_ </span> [https://monahton.github.io/portfolio/]


---


## Professional Experience

`October 2020 - Current`
__Postodoctoral Researcher | Senior Computational Biologist__, San Raffaele Telethon Institute for Gene Therapy (TIGET), Milan - Italy   
   - Designed and implemented genomics, transcriptomics and proteomics data analyses, pipelines, and algorithms for biomedical research
   - Devised experiments and supported the research in the following research groups: (i) Retrovirus‑host interactions and innate immunity to gene transfer
of *Prof. Anna Kajaste‑Rudnitski*, and (ii) Human hematopoietic development and disease modeling of *Prof. Andrea Ditadi*
   - Collaborate with leading researchers to support groundbreaking studies


`September 2021 - September 2023`
__Data Carpentry Lessons Maintainer__, The Carpentries organization, California - USA   
   - Maintained core lessons for the Data Carpentry program
   - Reviewed submissions, discussions of change and set standards for the organization as a whole


`October 2017 - September 2020`
__PhD Internship__, Institute of Molecular Genetics Luigi Luca Cavalli-Sforza, National Research Council, Pavia - Italy
   - Led a bioinformatics research project titled ’Characterization of Genomic and Splicing Features of Long Non‑Coding RNAs’ within the Computational Biology Unit supervised by *Prof. Silvia Bione*

  
`July 2017 - October 2017`
__ERASMUS+ Traineeship__, Polish Academy of Sciences, Poznan - Poland

- Executed an integrated wet‑lab/bioinformatics project to characterize miRNA features and functions.
- Gained hands‑on experience with computational methods and tools for small‑RNA data analysis under the supervision of *Prof. Jorge A. Pinto Paiva*


`November 2015 - July 2017`
__Masters Internship__, University of Pavia - Department of Biology and Biotechnology L.Spallanzani, Pavia - Italy

- Executed a wet-lab research project titled 'Influence of divalent cations on DNA methylation and the modulation of chromatin structure in an _in situ_ ultrastructural study' supervised by *Prof. Marco Biggiogera*

---

## Teaching Experience

`May 2025 - Current`
__Associated Partner__, MiCCrobioTackle - Leibniz Institute for Natural Product Research and Infection Biology, Jena - Germany  
   - Delivering hands-on bioinformatics training workshops to PhD students within the MICCROBIOTACKLE consortium.

`September 24 2024 - September 25 2024`
__Summer School Trainer__, EMERALD Internation PhD Program for Medical Doctors, Berlin - Germany  
   - Delivered a 2‑day workshop in 'Advanced Techniques for Data Visualisation using R Programming Language'.

`March 2024 - Current`
__Bioinformatics Trainer__, German Cancer Research Center (DKFZ), Heidelberg - Germany  
   - Organized and delivered PhD courses in the usage of R programming language for Bioinformatics analyses.

`January 2023 - Current`
__Bioinformatics Trainer__, Max Delbrück Center for Molecular Medicine (MDC), Berlin - Germany  
   - Bioinformatics workshops instructor and organizer
   - Training graduate and post-graduate biologists in Bioinformatics analyses

`September 2019 - Current`
__Software and Data Carpentry Instructor__, The Carpentries organization, California - USA 

- Teaching foundational coding and data science skills to researchers in worldwide workshops
- 30+ workshops delivered in Europe, USA and Saudi Arabia
- Selected workshops:
  - ***Harvard Stem Cell Institute***: Genomics workshop instructor: Variant calling workflow using AWS
  - ***United States Centers for Disease Control (CDC)***: Genomics workshop instructor: Variant calling workflow using AWS
  - ***Jackson Laboratory***: Genomics workshop instructor: Variant calling workflow using AWS
  - ***Genentech, A member of the Roche group***: Data Carpentry instructor: Data analysis and visualization in R
  - ***Max Delbrück Center for Molecular Medicine (MDC)***: Genomics workshop instructor: Variant calling workflow using AWS
  - ***Rutgers University***: Genomics workshop instructor: Project management/ Introduction to cloud computing
  - ***Saudi Food and Drug Authority (sFDA)***: Data analysis and visualization in R
  - ***Atlanta University Center***: Genomics workshop instructor: Variant calling workflow using AWS
  - ***George Washington University***: Genomics workshop instructor: Variant calling workflow using AWS
  - ***NIH Network of the National Library of Medicine***: Library Carpentry instructor: Databases and SQL
  - ***Florida International University***: Software Carpentry instructor: R for reproducible analysis

`October 2018 - September 2020`
__Bioinformatics Tutor__, University of Pavia, Pavia - Italy
   - Bioinformatics Course - Masters in Molecular Biology and Genetics
   - Training and practical sessions in bioinformatics tools


---

## Projects

`March 2024 - February 2025`
__EARLY__ ‑ _dEsigning an ARtificial inteLligence‑based sYstem_ , Univesity of Macerata  
  - A multidisciplinary project among medical doctors, engineers, lawyers and researchers for designing an __AI tool__ for improving diagnosis, treat‑
ment, and monitoring of patients with rare diseases preserving fundamental rights by law.


---


## Education

`2017 - 2020`
__University of Pavia,__
PhD in Genetics, Molecular and Cellular Biology

`2015 - 2017`
__University of Pavia,__
Masters Degree in Molecular Biology and Genetics

`2012 - 2015`
__Lebanese University,__
Bachelor Degree in Earth and Life Sciences


---


## Skills

_Bioinformatics Skills_

- Extensive experience in -omics data analysis: Expertise in genomics, transcriptomics, proteomics, and metabolomics data interpretation and integration
- Advanced Proficiency in High-Throughput Sequencing Data Analysis: In-depth analysis of RNA-seq, single-cell RNA-seq, ChIP-seq, and ATAC-seq datasets, including quality control, alignment, differential expression analysis, and peak calling
- Extensive experience in programming languages: R, Python, Bash, Perl, Julia
- Skilled in leveraging cloud platforms, particularly AWS, for large-scale data processing and storage solutions
- Proficient in applying machine learning algorithms and techniques to biological data for predictive modeling and pattern recognition
- Comprehensive knowledge of Windows 7/8/10, macOS, and Linux environments
- Expertise in Git, Bitbucket, Gitlab and GitHub for collaborative development, version tracking, and source code management
- Basic knowledge in SQL for querying and managing biological databases.
- Proficient in Microsoft Office Suite for professional documentation, data presentation, and communication.
- Image Analysis and Visualization Software: Skilled in using Adobe Photoshop, ImageJ, Cell F, and GraphPad for image analysis, scientific visualization, and figure preparation.
  
_Wet Laboratory Skills_

- Proficient in Microscopy utilization and samples preparation: electron microscopy (ZEISS EM 900), electron spectroscopic imaging, fluorescence microscopy, light microscopy (bright field, dark field, phase contrast, DIC)
- Proficient in molecular biology lab techniques: DNA/RNA extraction, PCR, gene cloning, protein extraction, western blot, ELISA
- Proficient in microbiology laboratory techniques: gram staining, bacterial cell culture, antibiotic specificity, bacterial identification
biochemical testing.
  
_Language Skills_

- Mother-tongue Language: Arabic
- Proficient in English (C1) and Italian (B2) Languages
- Novice in French (A2) and Polish (A2) Languages


---


## Awards

`2021`
James Taylor Fellowship, _2021 Galaxy Community Conference (GCC2021)_

`2019`
Best Poster Prize, _EMBL-EMBO Symposium: The Non-Coding Genome_

`2018`
Fellowship Grant, _AIRC Foundation_

`2018`
Travel Grant, _INTEROMICS_

`2017`
Fellowship Grant, _Besta Neurological Institute_

`2017`
ERASMUS+ fellowship, _University of Pavia_


---

## Student research supervision

`2020`
__Roasria S__ (BSc)

`2019`
__Salviati L__ (MSc);
__Nicastro A__ (MSc);
__Belotti G__ (MSc);

`2018`
__Celli L__ (MSc);
__Postiglione U__ (MSc);
__Baldrighi GN__ (BSc)

`2017`
__Monaco L__ (MSc)

---



## Publications

_Articles_

`2025`
Paillet J, Gaudeaux P, __Abou Alezz M__, Moirangthem RD, Cascione S, Corredera MM, Dolens AC, De Mulder K, Velghe I, Lavaert M, Vandekerckhove B, Noémie R, Corneau A, Sadek H, Rault P, Joshi A, De La Grange P, Staal F, Taghon T, Olivier N, Ditadi A, André I, Soheili TS. Ex vivo-generated lymphoid progenitors encompass both T cell and innate lymphoid cell fates. Front Immunol. 2025. doi: 10.3389/fimmu.2025.1617707

`2025`
Costa-Verdera H, Meneghini V, Fitzpatrick Z, __Abou Alezz M__, Fabyanic E, Hunag X, Dzhashiashvili Y, Ahiya A, Mangiameli E, Valeri E, Crivicich G, Piccolo S, Cuccovillo I, Caccia R, Kai Chan Y, Bertin B, Ronzitti G, Engel EA, Merelli I, Mingozzi F, Gritti A, Kuranda K, Kajaste-Rudnitski A. AAV vectors trigger DNA damage response-dependent pro-in ammatory signalling in human iPSC-derived CNS models and mouse brain. Nat Commun. 2025. doi: 10.1038/s41467-025-58778-3

`2024`
El Boustani M, Mouawad N, __Abou Alezz M__. AP3M2: A key regulator from the nervous system modulates autophagy in colorectal cancer. Tissue Cell. 2024. doi: 10.1016/j.tice.2024.102593

`2024`
Valeri E, Breggion S, Barzaghi F, __Abou Alezz M__, Crivicich G, Pagani I, Forneris F, Sartirana C, Costantini M, Costi S, Marino A, Chiarotto E, Colavito D, Cimaz R, Merelli I, Vicenzi E, Aiuti A, Kajaste‐Rudnitski A. A novel STING variant triggers endothelial toxicity and SAVI disease. J Exp Med. 2024. doi: 10.1084/jem.20232167

`2024`
Scarfò R, Randolph LN, __Abou Alezz M__, El Khoury M, Gersch A, Li Z, Luff S, Tavosanis A, Ramondo GF, Valsoni S, Cascione S, Didelon E, Passerini L, Amodio G, Brandas C, Villa A, Gregori S, Merelli I, Freund JN, Sturgeon C, Tavian M, Ditadi A. CD32 captures committed haemogenic endothelial cells during human embryonic development. Nat Cell Biol. 2024. doi: 10.1038/s41556‐024‐01403‐0

`2023`
Valeri E, Unali G, Piras F, __Abou Alezz M__, Pais G, Benedicenti F, Lidonnici MR, Cuccovillo I, Castiglioni I, Arévalo S, Spinozzi G, Merelli I, Behrendt R, Oo A, Kim B, Landau NR, Ferrari G, Montini E, Kajaste-Rudnitski A. Removal of innate immune barriers allows efficient transduction of quiescent human hematopoietic stem cells. Mol Ther. 2023. doi: 10.1016/j.ymthe.2023.11.020

`2023`
Unali G, Crivicich G, Pagani I, __Abou Alezz M__, Folchini F, Valeri E, Matafora V, Reisz JA, Giordano AMS, Cuccovillo I, Butta GM, Donnici L, D'Alessandro A, De Francesco R, Manganaro L, Cittaro D, Merelli I, Petrillo C, Bachi A, Vicenzi E, Kajaste-Rudnitski A. Interferon-inducible phospholipids govern IFITM3-dependent endosomal antiviral immunity. EMBO J. 2023. doi.org/10.15252/embj.2022112234


`2023`
Giordano AMS, __Abou Alezz M__, Merelli I, Kajaste-Rudnitski A. Protocol to differentiate monolayer human induced pluripotent stem cells into inflammatory responsive astrocytes. STAR Protoc. 2023. doi.org/10.1016/j.xpro.2023.102142

`2022`
Giordano AMS, Luciani M, Gatto F, __Abou Alezz M__, Begh C, Della Volpe L, Migliara A, Valsoni S, Genua M, Dzieciatkowska M, Frati G, Tahraoui-Bories J, Giliani SC, Orcesi S, Fazzi E, Ostuni R, D'Alessandro A, Di Micco R, Merelli I, Lombardo A, Reijns MAM, Gromak N, Gritti A, Kajaste-Rudnitski A. DNA damage contributes to neurotoxic inflammation in Aicardi-Gouti res syndrome astrocytes. J Exp Med. 2022. doi.org/10.1084/jem.20211121


`2020`
__Abou Alezz M__, Celli L, Belotti G, Lisa A and Bione S. GC-AG Introns Features in Long Non-coding and Protein-Coding Genes
Suggest Their Role in Gene Expression Regulation. Front. Genet. 2020 11:488. doi:10.3389/fgene.2020.00488

---

_Book Chapters_
  
`2024`
__Abou Alezz M__, Artificial Intelligence Bioinformatics Systems for the Diagnosis and Treatment of Rare Diseases: Integration and Implementation. In: Vulpiani G (ed.) Diritto, Intelligenza Artificiale e Medicina. Un dialogo interdisciplinare. Napoli: Edizioni Scientifiche Italiane. 2024. ISBN: 978-88-495-5722-0

---
  
_Software_

`2025`
__Abou Alezz M__. NatChat: Chatting with Nature Journals Current Issue using a local Language Model.
R-package version 1.1.0. https://monahton.github.io/NatChat. 2025. doi:10.5281/zenodo.15482472

`2025`
__Abou Alezz M__. GencoDymo2: Comprehensive Analysis of GENCODE Annotations and Splice Site Motifs.
R-package version 1.0.1. https://github.com/monahton/GencoDymo2. 2025. doi:10.5281/zenodo.15302316

`2020`
__Abou Alezz M__, Salviati L, Alfieri R, Bione S. GencoDymo: Data Extraction and Manipulation from the GENCODE Database.
R-package version 0.2.1. https://github.com/monahton/GencoDymo. 2020. doi:10.5281/zenodo.3605995

---



<p align="right">Last updated: 2025-05-31</p>


